cluster/repository/cluster_repository.go

```diff
@@ -85,7 +149,7 @@ func (m *GormClusterRepository) TableName() string {
 
 // TableName overrides the table name settings in Gorm to force a specific table name
 // in the database.
-func (m Cluster) TableName() string {
+func (c *Cluster) TableName() string {
```

Comment:
Do we really need pointer receiver here?

Reply:
right, I thought it would be necessary to have a pointer receiver on all functions on the `repository.Cluster`, but apparently not, so I'll remove it.

Reply:
10666f8